### PR TITLE
Add publishing information for PS repos

### PR DESCRIPTION
**Pull request by sdwheeler**
Add publishing information for PS repos

- Fixes [AB#125394](https://dev.azure.com/msft-skilling/cebd7ef5-4282-448b-9701-88c8637581b7/_workitems/edit/125394)

### CODE REVIEW

**Comment by sdwheeler** [comment]
@anamnavi Please review.

File: powershell-gallery/docs-conceptual/powershellget/supported-repositories.md [hunk 1]
```diff
@@ -142,9 +178,47 @@ GitHub Packages repositories:
   - The package is published as a private package. After publishing, you can use the GitHub
     interface to link it to the desired repository and change the visibility.
 
+### Publishing to GitHub Packages
+
+You can use the **Credential** or **ApiKey** parameter of the `Publish-PSResource` cmdlet to publish
+packages to a GitHub Package feed. You must create a personal access token (PAT) with the necessary
+scopes enabled. For more information on scopes and permissions, see
```
Comment (michaeltlombardi) [inline powershell-gallery/docs-conceptual/powershellget/supported-repositories.md:185]: We call out the specific scopes in the Azure Artifacts section, should we do the same here?

File: powershell-gallery/docs-conceptual/powershellget/supported-repositories.md [hunk 2]
```diff
@@ -215,6 +315,29 @@ repositories:
 - Publish packages to a feed
   - Returns a `403 (Forbidden)` error
 
+### Publishing to a MyGet feed
+
+You must use the **ApiKey** parameter of the `Publish-PSResource` cmdlet with personal access token
+(PAT) to publish packages to a MyGet feed. The value of the **ApiKey** parameter must be a plaintext
+PAT from your MyGet account that has read and write permission. For more information about creating
```
Comment (michaeltlombardi) [inline powershell-gallery/docs-conceptual/powershellget/supported-repositories.md:322]: Nit:
```suggestion
PAT from your MyGet account that has read and write permissions. For more information about creating
```

**Review by michaeltlombardi — APPROVED** [review]
One nit, one question, otherwise LGTM!

File: powershell-gallery/docs-conceptual/powershellget/supported-repositories.md (hunk 1 above)
Comment (sdwheeler) [reply powershell-gallery/docs-conceptual/powershellget/supported-repositories.md:185]: The scopes are in the linked document. The scopes are more granular (independent). If you only want to write, you don't need delete. So I left it to the reader to decide.

**Review by anamnavi — APPROVED** [review]
LGTM

**Pull request merged**